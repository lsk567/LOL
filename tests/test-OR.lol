bool h = true;
bool i = true;
bool j = false;

bool t1 = h OR i;
bool t2 = h OR j;

if (t1) {
	println("T OR T is T");
} else {
	println("T OR T is F");
}

if (t2) {
	println("T OR F is T");
} else {
	println("T OR F is F");
}


