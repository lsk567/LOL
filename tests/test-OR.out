T OR T is T
T OR F is T
